### PR TITLE
feat: add fixtures for products catalog

### PR DESCRIPTION
**Pull request by knop-k**
Added fixtures for products and added default photo for products

### CODE REVIEW

**Review by jacoor — CHANGES_REQUESTED** [review]
Hey
Please update readme with instructions on how to use it. 
WHen I try to loaddata I get below error

```
» ./manage.py loaddata fixtures_products.json
Traceback (most recent call last):
  File "/Users/jacek/Workspace/akademiaIT/Dshop/Dshop/./manage.py", line 22, in <module>
    main()
  File "/Users/jacek/Workspace/akademiaIT/Dshop/Dshop/./manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/Users/jacek/Library/Caches/pypoetry/virtualenvs/dshop--875GFOx-py3.11/lib/python3.11/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/Users/jacek/Library/Caches/pypoetry/virtualenvs/dshop--875GFOx-py3.11/lib/python3.11/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/jacek/Library/Caches/pypoetry/virtualenvs/dshop--875GFOx-py3.11/lib/python3.11/site-packages/django/core/management/base.py", line 412, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/jacek/Library/Caches/pypoetry/virtualenvs/dshop--875GFOx-py3.11/lib/python3.11/site-packages/django/core/management/base.py", line 458, in execute
    output = self.handle(*args, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jacek/Library/Caches/pypoetry/virtualenvs/dshop--875GFOx-py3.11/lib/python3.11/site-packages/django/core/management/commands/loaddata.py", line 102, in handle
    self.loaddata(fixture_labels)
  File "/Users/jacek/Library/Caches/pypoetry/virtualenvs/dshop--875GFOx-py3.11/lib/python3.11/site-packages/django/core/management/commands/loaddata.py", line 171, in loaddata
    connection.check_constraints(table_names=table_names)
  File "/Users/jacek/Library/Caches/pypoetry/virtualenvs/dshop--875GFOx-py3.11/lib/python3.11/site-packages/django/db/backends/sqlite3/base.py", line 275, in check_constraints
    raise IntegrityError(
django.db.utils.IntegrityError: Problem installing fixtures: The row in table 'products_catalogue_category' with primary key '1' has an invalid foreign key: products_catalogue_category.ceneo_category_id contains a value '1305' that does not have a corresponding value in products_catalogue_ceneocategory.id.
```

File: README.md [hunk 1]
```diff
@@ -85,6 +85,11 @@ python manage.py migrate
 python manage.py runserver
 ```
 
+Load data from fixtures:
+```
+python manage.py loaddata apps\products_catalogue\fixtures\fixtures_products.json
```
Comment (jacoor) [inline README.md]: Actually, at least on Mac, it works only with 
`python manage.py loaddata fixtures_products.json`

Could you check if windows is ok too? If so, please update.

**Review by jacoor — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (knop-k) [reply README.md]: Yes, it works on windows too. I think Django itself has to search by its structure for this file.

**Review by jacoor — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jacoor) [reply README.md]: It's supposed to search "fixtures" directory in apps. Thanks for change.

**Pull request merged**